molecule/with-python/molecule.yml

```diff
@@ -0,0 +1,23 @@
+---
+dependency:
+  name: galaxy
+  options:
+    role-file: ./molecule/with-python/requirements.yml
+driver:
+  name: docker
+platforms:
+  - name: instance-rsc-with-python
+    image: "geerlingguy/docker-${MOLECULE_DISTRO:-ubuntu2004}-ansible:latest"
+    command: ${MOLECULE_DOCKER_COMMAND:-""}
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    published_ports:
+      - "0.0.0.0:3940:3939/tcp"
```

Comment:
Why you changed the ports here?

Reply:
Both scenarios had `0.0.0.0:3939:3939/tcp`, so running them both (at the same time) results in error (one of them fails with information that port `3939` is already allocated). So I just changed the port for `with-python` scenario.